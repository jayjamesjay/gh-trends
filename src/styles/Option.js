import styled from 'styled-components';

const Option = styled.option`
  font-size: 1.2rem;
  text-transform: uppercase;
  line-break: loose;
  text-align: center;
`;

export default Option;
