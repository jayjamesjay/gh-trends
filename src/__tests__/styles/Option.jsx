import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';
import Option from '../../styles/Option';

it('renders default <Option/>', () => {
  const { asFragment } = render(<Option />);
  expect(asFragment()).toMatchSnapshot();
});
