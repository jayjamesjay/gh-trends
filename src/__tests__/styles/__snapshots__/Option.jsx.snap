// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default <Option/> 1`] = `
<DocumentFragment>
  .c0 {
  font-size: 1.2rem;
  line-break: loose;
  text-align: center;
  text-transform: uppercase;
}

<option
    class="c0"
  />
</DocumentFragment>
`;
